esignet-insurance.properties

```diff
@@ -336,7 +340,7 @@ mosip.kernel.keymanager.hsm.keystore-type=PKCS11
 mosip.kernel.keymanager.hsm.config-path=/config/softhsm-application.conf
 # Passkey of keystore for PKCS11, PKCS12
 # For Offline & JCE proer can be left blank. JCE password use other JCE specific properties.
-mosip.kernel.keymanager.hsm.keystore-pass=${softhsm.esignet.mock.security.pin}
+mosip.kernel.keymanager.hsm.keystore-pass=${softhsm.esignet.security.pin}
```

Comment:
if the esignet-insurance was running with the same security pin, I don't think we should change this.